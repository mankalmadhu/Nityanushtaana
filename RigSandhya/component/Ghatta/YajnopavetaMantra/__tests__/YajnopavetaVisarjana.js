import React from 'react';
import ಯಜ್ಞೋಪವೀತವಿಸರ್ಜನ from '../YajnopavetaVisarjana';
import Renderer from 'react-test-renderer';
 

jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಯಜ್ಞೋಪವೀತವಿಸರ್ಜನ", () => {
   expect(Renderer.create(
    <ಯಜ್ಞೋಪವೀತವಿಸರ್ಜನ />
  ).toJSON()).toMatchSnapshot();

});