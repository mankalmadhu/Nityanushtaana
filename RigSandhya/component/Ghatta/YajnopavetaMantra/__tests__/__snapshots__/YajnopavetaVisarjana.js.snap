// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಯಜ್ಞೋಪವೀತವಿಸರ್ಜನ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ವಿಸರ್ಜನ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಉಪವೀತ೦ ಭಿನ್ನತ೦ತು೦ ಜೀರ್ಣ ಕಶ್ಮಲದೊಷಿತ೦ । ವಿಸೃಜಾಮಿ ಯಶೋಬ್ರಹ್ಮನ್ ವರ್ಚೋ ದೀರ್ಘಾಯುರಸ್ತು ಮೇ ॥ ಸಮುದ್ರ೦ ಗಚ್ಛಸ್ವಾಹಾ ॥
  </ವಿಷಯ>
</View>
`;
